monero-wallet-generator.html

```diff
@@ -11577,9 +11600,11 @@ <h5>Checking a git commit's signature</h5>
   view_key_widget = document.getElementById("view_key_widget");
   mnemonic_widget = document.getElementById("mnemonic_widget");
   if (priv_mnemonic_qr_widget.style.display=="block") {
+    shouldDimoutPage(false)
     hide_qr();
```

Comment:
Looks like the call to shouldDimoutPage (badly named btw, sounds like a function that returns whether it shold be dimmed out, not a procedure to do so) should be called fro hide_qr directly. Smaller changes and no way to forget a call then.